usaspending_api/awards/v2/views/idvs/activity.py

```diff
@@ -78,11 +80,49 @@
     where   {award_id_column} = {award_id}
 """)
 
+COUNT_ACTIVITY_HIDDEN_SQL = SQL("""
+    with gather_award_ids as (
+        select  award_id,
+                false grandchild
+        from    parent_award
+        where   {award_id_column} = {award_id}
+        union all
+        select  cpa.award_id,
+                true grandchild
+        from    parent_award ppa
+                inner join parent_award cpa on cpa.parent_award_id = ppa.award_id
+        where   ppa.{award_id_column} = {award_id}
+    )
+    select
+        count(ca.id) rollup_contract_count
```

Comment:
So, believe it or not, there's a very minor efficiency gain with `count(*)` over counting a specific column.  When counting a column, the database has to check it for `null`.  Since this column can never be null in this query, it's more efficient to use `count(*)`.